### ISSUE TITLE
Plural strings are not extracted from js files

### ISSUE THREAD

**Issue by Chouby** [wp-cli/i18n-command#166]
## Bug Report

- [x] Yes, I reviewed the [contribution guidelines](https://make.wordpress.org/cli/handbook/contributing/).
- [x] Yes, more specifically, I reviewed the guidelines on [how to write clear bug reports](https://make.wordpress.org/cli/handbook/bug-reports/).

**Describe the current, buggy behavior**
The command wp i18n make-pot doesn't correctly extract plural strings in js files.

**Describe how other contributors can replicate this bug**
- Create a js file with the following code:
```js
if ( !! posts.length ) {
	this.props.debouncedSpeak(
		sprintf(
			/* translators: accessibility text. %d is a number of posts. */
			_n(
				'%d result found, use up and down arrow keys to navigate.',
				'%d results found, use up and down arrow keys to navigate.',
				posts.length,
				'polylang-pro'
			),
			posts.length
		),
		'assertive' );
} else {
	/* translators: accessibility text */
	this.props.debouncedSpeak( __( 'No results.', 'polylang-pro' ), 'assertive' );
}
```
Then run the command:
`wp i18n make-pot . test.pot --domain=polylang-pro`

**Describe what you would expect as the correct outcome**

The message `Success: POT file successfully generated!` is correctly displayed but the generated test.pot file includes only the singular string `No results`. It doesn't include the plural string. Strangely if I replace the 3rd parameter by a hardcoded int like this:
```js
if ( !! posts.length ) {
	this.props.debouncedSpeak(
		sprintf(
			/* translators: accessibility text. %d is a number of posts. */
			_n(
				'%d result found, use up and down arrow keys to navigate.',
				'%d results found, use up and down arrow keys to navigate.',
				1,
				'polylang-pro'
			),
			posts.length
		),
		'assertive' );
} else {
	/* translators: accessibility text */
	this.props.debouncedSpeak( __( 'No results.', 'polylang-pro' ), 'assertive' );
}
```
Then the generated pot file correctly includes the plural string.

**Let us know what environment you are running this on**

```
OS:	Linux 4.15.0-48-generic #51-Ubuntu SMP Wed Apr 3 08:28:49 UTC 2019 x86_64
Shell:	/bin/bash
PHP binary:	/opt/lampp/bin/php-7.1.21
PHP version:	7.1.21
php.ini used:	/opt/lampp/etc/php.ini
WP-CLI root dir:	phar://wp-cli.phar/vendor/wp-cli/wp-cli
WP-CLI vendor dir:	phar://wp-cli.phar/vendor
WP_CLI phar path:	/home/fred/Documents/Web/test-i18n
WP-CLI packages dir:	
WP-CLI global config:	
WP-CLI project config:	
WP-CLI version:	2.2.0
```

**Comment by swissspidy** [comment]
Thanks for your report!

That sounds odd. I'll try to investigate this a bit and perhaps add a failing test first to verify this.

**Comment by manooweb** [comment]
for information extract works with the babel plugin : @wordpress/babel-plugin-makepot

**Comment by swissspidy** [comment]
Here's currently the only place where we test this scenario:

https://github.com/wp-cli/i18n-command/blob/af8143381e6a96555b533471e8f9680d755d42d9/features/makepot.feature#L1510-L1559

**Comment by swissspidy** [comment]
Duplicate of #225. This will be fixed by #258
